wordpress-mobile/WordPress-Android
# Media picker edit button tweaks

After having merged the #12984, while testing another PR I noticed a couple of tweaks we could add to the Media Picker edit button that I didn't notice during review (sorry @khaykov 🙇 ).

This PR do the following:

1. Adds a tooltip to the edit button when long pressed
2. Review the margin of the edit icon badge to account for the fact that we reduced the layout from 64dp to 48dp

### To test
- In media picker long press the edit icon and notice you get the tooltip

| ![image](https://user-images.githubusercontent.com/47797566/94008113-b7e52b00-fda2-11ea-888b-8d7d9b039368.png) |
|---|

- In media picker select some images and note the badge is correctly aligned on right top of the icon

| ![image](https://user-images.githubusercontent.com/47797566/94008177-d77c5380-fda2-11ea-90f1-d084a5ca454d.png) |
|---|


PR submission checklist:
